go.mod

```diff
@@ -47,6 +46,7 @@ require (
 	github.com/stretchr/testify v1.3.0
 	github.com/texttheater/golang-levenshtein v0.0.0-20180516184445-d188e65d659e
 	github.com/uber/jaeger-client-go v2.15.0+incompatible
+	github.com/y0ssar1an/q v1.0.6 // indirect
```

Comment:
Do we actually need this, or was this a debugging artifact?

Reply:
Good catch! That was for debugging